packages/utils-compat-data/tests/browsers.ts

```diff
@@ -219,3 +224,8 @@ test('Includes accurate details', (t) => {
         versionAdded: '15'
     }, 'Before -webkit- support');
 });
+
+test('Returns the mdn url', (t) => {
+    t.is(getUnsupportedBrowsers(keyframes, '', ['opera 12'], 'keyframes')!.mdnUrl, 'https://developer.mozilla.org/docs/Web/CSS/@keyframes', 'Right mdn url before first unprefixed support');
+    t.is(getUnsupportedBrowsers(maxContent, '', ['firefox 65'], 'max-content', width)!.mdnUrl, 'https://developer.mozilla.org/docs/Web/CSS/width');
```

Comment:
Nit: I think we should start using optional chaining for these situations now that it's available. Then if we're ever wrong we'll get `undefined` instead of an exception. Not as big of a deal in tests, but definitely in our other code. What do you think?
```suggestion
    t.is(getUnsupportedBrowsers(maxContent, '', ['firefox 65'], 'max-content', width)?.mdnUrl, 'https://developer.mozilla.org/docs/Web/CSS/width');
```